### DISCUSSION TITLE
[Build Error ]Failed to list versions for com.netflix.runtime:health-guice.

### DISCUSSION THREAD

**Discussion by sakim-hooda** [Netflix/conductor#2638] [Q&A]
* What went wrong:
A problem occurred configuring project ':conductor-server'.
> Could not resolve all dependencies for configuration ':conductor-server:runtimeCopy'.
   > Could not resolve com.netflix.runtime:health-guice:1.1.+.
     Required by:
         project :conductor-server
      > Failed to list versions for com.netflix.runtime:health-guice.
         > Unable to load Maven meta-data from https://dl.bintray.com/netflixoss/oss-candidate/com/netflix/runtime/health-guice/maven-metadata.xml.
            > Could not get resource 'https://dl.bintray.com/netflixoss/oss-candidate/com/netflix/runtime/health-guice/maven-metadata.xml'.
               > Could not GET 'https://dl.bintray.com/netflixoss/oss-candidate/com/netflix/runtime/health-guice/maven-metadata.xml'. Received status code 502 from server: Bad Gateway

* Try:

**Comment by aravindanr** [comment]
Could you give us more information about which version you are trying to build?

**Comment by devinderpal** [comment]
+1. I am also facing this issue in building 2.31.7. Any fix?

**Comment by noamlaf** [comment]
Same for 2.31.0

**Comment by kushalhalder** [comment]
Same For 2.25.11

**Reply by aravindanr** [reply]
2.25.11 is a really old version and not supported.  Please upgrade to 3.x or at least to 2.31. We have a fix coming up for 2.31, but it's likely that 2.31 won't be supported for long.

**Comment by aravindanr** [comment]
Fix is in this PR, https://github.com/Netflix/conductor/pull/2720

**Comment by aravindanr** [comment]
@sakim-hooda @devinderpal @noamlaf @kushalhalder We are undertaking an effort to know more about conductor users. Appreciate if you could add your company details to [WHOSUSING.md](https://github.com/Netflix/conductor/blob/main/WHOSUSING.md). Example PR, #2707
